.github/workflows/build.yml

```diff
@@ -0,0 +1,34 @@
+name: Build Action
+
+on:
+  push:
+    branches:
+      - main
+      - releases/*
+      - v2-preview
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Node.JS 12
+        uses: actions/setup-node@v2
+        with:
+          node-version: 12.x
```

Comment:
Actiosn confirmed for single storage have successfully integrated development with app processing on my IDE  with significant running actions.